src/shared/tpm2-util.h

```diff
@@ -126,11 +126,11 @@ static inline bool TPM2_PCR_MASK_VALID(uint64_t pcr_mask) {
 #define TPM2_ALG_RSA 0x1
 #endif
 
-const char *tpm2_pcr_bank_to_string(uint16_t bank);
-int tpm2_pcr_bank_from_string(const char *bank);
+const char *tpm2_hash_alg_to_string(uint16_t bank);
+int tpm2_hash_alg_from_string(const char *bank);
```

Comment:
rename the parameter too?

Reply:
oops! changed it in the .c but forgot in the header :) updated in last push, thanks!